scripts/release.sh

```diff
@@ -34,18 +43,22 @@ echo "CHANGELOG:"
 echo "$CHANGELOG"
 echo ""
 
-UPLOADS_URL=$( \
-  echo '{}' | \
+RELEASE_ARGS=$(echo '{}' | \
     jq -Mr \
       --arg version "$VERSION" \
       --arg changelog "$CHANGELOG" \
       '. + {
         tag_name: $version,
         name: $version,
         body: $changelog,
-      }' | \
-    curl --user "$GITHUB_USER:$GITHUB_TOKEN" -X POST --data @- \
-      "https://api.github.com/repos/$GITHUB_REPO/releases" | \
+      }')
+
+RELEASE_URL="https://api.github.com/repos/$GITHUB_REPO/releases"
+RELEASE_OUT=$(echo "$RELEASE_ARGS" | curl --user "$GITHUB_USER:$GITHUB_TOKEN" -X POST --data @- "$RELEASE_URL")
+
+echo "Release to $RELEASE_URL got:\n$RELEASE_OUT"
```

Comment:
For multi-line output, use separate echoes if possible.

```suggestion
echo "Release to $RELEASE_URL got:"
echo "  $RELEASE_OUT"
```

It's slightly more readable and you don't have to deal with issues like,

```
$ bash
$ echo "foo\nbar"
foo\nbar
$ echo -e "foo\nbar"
foo
bar
```